tests/E2E/test/campaigns/regular/03_create_product.js

```diff
@@ -19,6 +21,7 @@ scenario('Create "Product"', () => {
     test('should open the browser', () => client.open());
     test('should login successfully in the Back Office', () => client.signInBO(AccessPageBO));
   }, 'common_client');
+  welcomeScenarios.findAndCloseWelcomeModal();
```

Comment:
so this was necessary to fix the tests, could there be other places where this could be needed? how can we identify them?
(also c.f. my first comment, this context parameter could be `product`)

Reply:
not sure I got it all ^^
but do we keep the default parameter 'order' here then?

Reply:
Based on the previous script calling this code, yep.

Reply:
ok the it's good for me! ;)